Gemfile

```diff
@@ -60,10 +60,15 @@ group :production do
   gem "newrelic_rpm"
 end
 
+group :development, :test, :staging do
+  gem 'factory_bot_rails'
+  gem "database_cleaner"
+  gem 'faker'
+end
```

Comment:
Added these so that `:staging` can utilize `database_cleaner` and `faker` for the cleaning & reseeding of the DB in staging.